VimeoNetworking/Sources/Models/VIMConnection.m

```diff
@@ -90,4 +91,11 @@ - (BOOL)canPost
     return (self.options && [self.options containsObject:@"POST"]);
 }
 
+- (void)setNotifications:(nonnull NSDictionary *)dic
```

Comment:
Can we handle this like we handle the VOD stuff (do this in the `didFinishMapping` method rather than in a separate `setNotifications` method)? I think that will be more clear the the existing comment from Nicole will apply to both.